components/generic/OptimisedImage.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <span>
+    <b-img-lazy
+      v-if="lazy"
+      :src="optimisedSrc"
+      :width="optimisedWidth"
+      :height="optimisedHeight"
```

Comment:
Could you set this to blank-width and blank-height, so CSS can still do some resizing if needed.